cookbooks/hdfsdu/recipes/hdfsdu_deploy.rb

```diff
@@ -0,0 +1,106 @@
+# Cookbook Name : hdfsdu
+# Recipe Name   : hdfsdu_deploy.rb
+# Description   : Download, configure and start hdfsdu
+
+Chef::Resource.send(:include, Hdfsdu::Helper)
```

Comment:
From a discussion I had with Noah Kantrowitz I believe you'll want to use `:extends` instead of `:include`.